test/TuviellaToken.spec.js

```diff
@@ -1,22 +1,24 @@
 const Web3 = require('web3');
 const TuviellaToken = artifacts.require('TuviellaToken');
-
+var { MIN_ABI } = require('./utils');
 const web3 = new Web3('http://localhost:8545');
 
 contract('TuviellaToken', (accounts) => {
   const masterChef = accounts[0];
   const faucet = accounts[1];
   let instance;
   let contractAddress;
+  let contract;
 
   before(async () => {
     instance = await TuviellaToken.deployed(masterChef, faucet);
     contractAddress = instance.address;
+    contract = new web3.eth.Contract(MIN_ABI, contractAddress);
   });
 
-  it('Should divide initial funds in masterChef and faucet', () => {
-    const masterChefBalance = instance.balanceOf(masterChef);
-    const faucetBalance = instance.balanceOf(faucet);
+  it('Should divide initial funds in masterChef and faucet', async () => {
+    const masterChefBalance =   await contract.methods.balanceOf(masterChef).call();
```

Comment:
si se usa la instancia creada con deployed, no hace falta el abi

Reply:
https://ethereum.stackexchange.com/questions/34302/instance-balanceof-call-is-not-a-function/34371#34371